queue_services/entity-filer/src/entity_filer/filing_processors/correction.py

```diff
@@ -47,9 +47,7 @@ def process(correction_filing: Filing, filing: Dict, filing_meta: FilingMeta, bu
     )
 
     corrected_filing_type = filing['correction']['correctedFilingType']
-    # added CP, change of directors / change of address for CP is allowed
-    if business.legal_type in ['SP', 'GP', 'BC', 'BEN', 'CC', 'ULC', 'CP'] and \
-            corrected_filing_type != 'conversion':
+    if corrected_filing_type != 'conversion':
```

Comment:
not sure if we need this if as well. any thoughts @argush3?

Reply:
I think we can remove L50 now.  We previously didn't allow corrections of corrections but we allowed it via https://github.com/bcgov/entity/issues/16858.  think we forgot to update the filter for this.

Reply:
FWIW, the UI doesn't allow correction of a record conversion. That's a function of a FF and allowable actions. This check here might be good for API safety...?

Reply:
Since we are correcting latest data checking corrected_filing_type make no sense

Reply:
FWIW2, there are other filings that we don't allow correcting.

Ref: https://github.com/bcgov/business-filings-ui/blob/29cc2f6b204f3755ad13eafd45aa4e524337b071/src/components/Dashboard/FilingHistoryList/HeaderActions.vue#L136

Reply:
actually, after looking at https://app.zenhub.com/workspaces/entities---david-65af11a89e89f5043c2911f6/issues/gh/bcgov/entity/16858 more closely, it seems like the change was related to allow local correction of conversion filing. 

The L50 check is used to make sure the correction stays in `PENDING_CORRECTION` status instead of actually getting processed by the filer like other filing types.

So I think we should keep L50 as is.

Reply:
> Since we are correcting latest data checking corrected_filing_type make no sense

So nothing is done with corrected_filing_type? It's obsolete? (Makes sense to me.)